src/manager.rs

```diff
@@ -174,15 +227,64 @@ impl NetworkManager {
         self.property(path, NM_DEVICE_INTERFACE, "Real")
     }
 
+    pub fn activate_device(&self, path: &String) -> Result<(), String> {
+        try!(self.call_with_args(NM_SERVICE_PATH,
+                                 NM_SERVICE_INTERFACE,
+                                 "ActivateConnection",
+                                 vec![&try!(Path::new("/")) as &RefArg,
+                                      &try!(Path::new(path.as_str())) as &RefArg,
+                                      &try!(Path::new("/")) as &RefArg]));
+
+        Ok(())
+    }
+
+    pub fn disconnect_device(&self, path: &String) -> Result<(), String> {
+        try!(self.call(path, NM_DEVICE_INTERFACE, "Disconnect"));
+
+        Ok(())
+    }
+
+    pub fn get_device_access_points(&self, path: &String) -> Result<Vec<String>, String> {
```

Comment:
No, it is just like the other public methods of the NetworkManager struct. Or I am missing something?